src/Client/app/(admin)/exams/create/page.tsx

```
@@ -1,270 +1,414 @@
+    const handleSaveExam = async (e: React.FormEvent) => {
+        e.preventDefault();
+    
+        if (!date) {
+            toast.error("Please select a date");
+            return;
+        }
+        try {
+            const formatDateTimeToUTC = (timeString: string | undefined, selectedDate: CalendarDate | null): string => {
+                if (!timeString || !selectedDate) return "";
+                const [hoursStr, minutesStr] = timeString.split(":");
+                const hours = parseInt(hoursStr, 10);
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)